kaldb/src/main/java/com/slack/kaldb/preprocessor/PreprocessorRateLimiter.java

```diff
@@ -150,6 +151,17 @@ public Predicate<String, Trace.Span> createRateLimiter(
         return false;
       }
 
+      if (value.getTimestamp() <= 0) {
```

Comment:
I'm not a huge fan of this sitting here - it's not really part of rate limiting; If we intend to drop this due to a missing timestamp within the preprocessor, in the value mapper would be a better fit for it.

Reply:
actually you are right! The value mapper is before the rate limiter so lemme move it there